cli/wallet/wallet.go

```diff
@@ -657,21 +718,28 @@ func createWallet(ctx *cli.Context) error {
 
 func readAccountInfo() (string, string, error) {
 	rawName, _ := input.ReadLine("Enter the name of the account > ")
+	name := strings.TrimRight(rawName, "\n")
```

Comment:
`input.ReadLine` removes trailing newline. So do we need to trim?

Reply:
Fixed.